### PR TITLE
#2290, 2407

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/2290 - [studio-ui] Update UI to add a global menu #2290

### CODE REVIEW

File: static-assets/ng-views/adminConsole.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="admin-console-view col-lg-10 col-lg-offset-1">
+
+    <header class="screen-title">
+        <h1>Admin Console</h1>
```
Comment (sumerjabri) [inline static-assets/ng-views/adminConsole.html]: Can we rename this file and these headings to Global Menu instead?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: static-assets/ng-views/adminConsole.html (hunk 1 above)
Comment (vestrada) [reply static-assets/ng-views/adminConsole.html]: Done

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Can we also rename the file adminConsole.html to global menu?

File: static-assets/ng-views/adminConsole.html (hunk 1 above)
Comment (vestrada) [reply static-assets/ng-views/adminConsole.html]: Done

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**